src/components/notifications/46elks.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div class="form-text">
+        <p>
+            You find the API-Username and API-password at <a href="https://46elks.com/account">https://46elks.com/account</a>
+        </p>
+    </div>
+    <div class="mb-3">
+        <label for="ElksUsername" class="form-label">{{ $t("elksUsername") }}</label>
+        <input id="ElksUsername" v-model="$parent.notification.elksUsername" type="text" class="form-control" required>
+    </div>
+
+    <div class="mb-3">
+        <label for="ElksPassword" class="form-label">{{ $t("elksPassword") }}</label>
+        <input id="ElksPassword" v-model="$parent.notification.elksAuthToken" type="text" class="form-control" required>
```

Comment:
Please use a `<HiddenInput ..` instead to make it clear to our users that this value is sensitive